File: app/src/main/java/de/tum/in/tumcampus/models/CalendarRow.java
package de.tum.in.tumcampus.models;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@SuppressWarnings("UnusedDeclaration")
@Root(name = "event")
public class CalendarRow {
	@Element(required = false)
	private String description = "";

	@Element(required = false)
	private String dtend = "";

	@Element(required = false)
	private String dtstart = "";

	@Element(required = false)
	private Geo geo;

	@Element(required = false)
	private String location = "";

	@Element(required = false)
	private String nr = "";
	// <org_kennung_betreut>TUEIEDA</org_kennung_betreut>

	@Element(required = false)
	private String status = "";
	// <org_name_betreut>Lehrstuhl für Entwurfsautomatisierung (Prof.
	// Schlichtmann)</org_name_betreut>

	@Element(required = false)
	private String title = "";

	@Element(required = false)
	private String url = "";

	// <org_nr_betreut>15393</org_nr_betreut>

	public String getDescription() {
		return description;
	}

	public String getDtend() {
		return dtend;
	}

	public String getDtstart() {
		return dtstart;
	}

	public Geo getGeo() {
		return geo;
	}

	public String getLocation() {
		return location;
	}

	public String getNr() {
		return nr;
	}

	public String getStatus() {
		return status;
	}

	public String getTitle() {
		return title;
	}

	public String getUrl() {
		return url;
	}

    public void setDtstart(String dtstart) {
        this.dtstart = dtstart;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public void setGeo(Geo geo) {
        this.geo = geo;
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/CalendarRow.kt
package de.tum.`in`.tumcampusapp.models.tumo

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "event")
data class CalendarRow(
        @field:Element(required = false) var description: String = "",
        @field:Element(required = false) var dtend: String = "",
        @field:Element(required = false) var dtstart: String = "",
        @field:Element(required = false) var geo: Geo? = null,
        @field:Element(required = false) var location: String = "",
        @field:Element(required = false) var nr: String = "",
        @field:Element(required = false) var status: String = "",
        @field:Element(required = false) var title: String = "",
        @field:Element(required = false) var url: String = "")
